### DISCUSSION TITLE
Operation with motion sensor.

### DISCUSSION THREAD

**Discussion by kivanc87** [RavenSystem/esp-homekit-devices#2014] [MEPLHAA Scripts]
Hello, I will be glad if you can help me with this. Can I run this code with motion sensor?

{"c":{"io":[[[0],6],[[2],7]]},"b":[{"g":0,"t":5}],"a":[{"t":30,"g":[2],"b":[{"g":0}]}]}

**Comment by kivanc87** [comment]
@xrust83 plase

**Reply by xrust83** [reply]
this is the code for the light bulb

**Reply by kivanc87** [reply]
yes i want it to turn off flashing with motion sensor like button. So I want to do button conversations by feeling. I would be glad if you help. @xrust83
